docs/dev/release.md

```diff
@@ -154,8 +164,18 @@ If script successfully finishes, you should be able to see new tag (e.g. 1.2.0)
 Then go the GitHub and create a PR
 from the `mongodb-fork` repository to https://github.com/redhat-openshift-ecosystem/certified-operators (`origin`).
 
+Before posting the PR there are manual changes you need to make:
+
+1. Ensure to add the `quay.io/` prefix in all Operator image references.
+1. Add missing a `com.redhat.openshift.versions: "v4.8"` line at the end of `metadata/annotations.yaml`.
+1. Ensure all image references, including `containerImage`, do NOT use the version *tag*. They **should only use the SHA of the AMD image**, NEVER the multi arch SHA.
+1. Add missing `spec.relatedImages` section in certified pinning all the images per architecture.
```

Comment:
What is "certified pinning" and what content shoidl `relatedImages` have?

Reply:
Yeah that wording is not very clear I will revisit it...

This is an example of ´relatedImages´:

´´´yaml
spec:
  relatedImages:
    - name: mongodb-atlas-kubernetes-operator-arm64
      image: quay.io/mongodb/mongodb-atlas-kubernetes-operator@sha256:f13541f1d63c9f9b959cad82823fcc9aeb9ac50f94339f0ce606affbccf1c4d0
    - name: mongodb-atlas-kubernetes-operator-amd64
      image: quay.io/mongodb/mongodb-atlas-kubernetes-operator@sha256:ec4cb2fc9e32c88a7372be51971a5e24a1c87251f68269423e553e2febf9d4eb
´´´

Basically certified does not seem to understand multi-arch images, so you need to specify each of the hashes of each of the architectures you support here. Elsewhere, when a single image ref is required, we tend to use the AMD one by default.